Update book (#83)
nomoixyz/vulcan: docs/src/modules/fs.md

# Fs

Provides utilities to interact with the filesystem. In order to use this module the

> `fs_persmissions` setting must bet correctly set on `foundry.toml`.

Reviewer: ```suggestion
`fs_permissions` setting must be set correctly in `foundry.toml`.
```

```solidity
import { Test, fs } from "vulcan/test.sol";

contract TestMyContract is Test {
    function testMyContract() external {
		// Write a string to a file
        fs.write("test.txt", "Hello World");

		// Write bytes to a file

> 		fs.writeBirnary("test.bin", abi.encodeWithSignature("test(uint256)", 1e18));

Reviewer: ```suggestion
		fs.writeBinary("test.bin", abi.encodeWithSignature("test(uint256)", 1e18));
```
